src/libc/src/math.c

```diff
@@ -3,3 +3,12 @@
 // TODO: External to OWI
 double sin(double x) { return x; }
 double cos(double x) { return x; }
+double floor(double x) { return x; }
+double round(double x) { return x; }
+float powf(float x, float y) { return x; }
+float floorf(float x) { return x; }
+double ceil(double x) { return x; }
+double sqrt(double x) { return x; }
+float roundf(float x) { return x; }
+float log2f(float x) { return x; }
+float sqrtf(float x) { return x; }
```

Comment:
@filipeom I believe we could map many of those directly to corresponding Wasm instructions, right? I'm thinking of stuff like `{f32,f64}.{floor,ceil,sqrt}`.